test/unit/src/server/scripts/t_poller.js

```diff
@@ -139,7 +141,18 @@ describe('Poller script helpers', function() {
       });
 
       it('gets checked and built if changed', async () => {
-        const threshold = conf.get('BUILD_TRIGGER_THRESHOLD');
+        // TODO: Exotic dance to re-import `pollRepositories` with the
+        // `POLLER_REQUEST_BUILD` configuration option enabled.
+        const requireWithMockConfig = requireWithMockConfigHelper.bind(
+          null,
+          path.resolve(__dirname, '../../../../../src/server/scripts/poller'),
+          '../helpers/config'
+        );
+        const pollRepositoriesMock = requireWithMockConfig({
+          POLLER_REQUEST_BUILD: true
```

Comment:
There is a typo here (missing S in `POLLER_REQUEST_BUILDS`) which causes this test to fail.